rfc/2022/2022-07-13 - Platform Wide Documentation vs Component Specific Documentation Locations.md

```diff
@@ -0,0 +1,76 @@
+    1. Examples include:
+        1. Documentation that is internal to a VFS team
+        1. Highly technical documentation about an infrastructure resource
+        1. Documentation for VA APIs and downstream external services
+1. **A clear relationship between the Platform Console and the Platform Website.** We propose that the Console and Website contain numerous references to each other. For example, if a component in the Console catalog has documentation that is relevant to all VFS users, it should have a link to Platform Website documentation on the topic. If the Platform Website has a page that mentions technical documentation that not all VFS teams need, it should link to the relevant documentation in the Console catalog. 
+1. **Storing documentation close to the software project/component.** Documentation for a  component should be stored in the same repo as the the component (where possible) or in a central documentation location, and then ingested into the Console catalog so it easy to find.
+1. **A self-service process for adding documentation.** Teams who own software components, services, and resources will register these entities in the Console catalog. They will include references to their technical documentation and spell out both upstream and downstream dependencies. They may also create catalog entries for dependencies (and their documentation) that are external to the VA.
+
+## Potential Benefits
+
+Possible benefits of a centralized system for component-specific technical documentation include these:
+
+* VFS and Platform teams will more easily maintain documentation about their software projects for themselves or others to reference later.
+* VFS teams will find it easier to document information about downstream systems that are external to the VA.
+* New VFS and Platform team members will more easily be able to learn about the software projects that their team maintains.
+* VFS and Platform team members will be able to more easily find and discover documentation about any VFS and Platform entity.
+* Each component’s source code and documentation will be easier to find and less likely to be lost than if they were stored in diverse locations.
+* With connections between component and their documentation better maintained, it will be easier to care for each software component, its documentation, and systems that depend upon the component.
+* With each component’s documentation stored close its source code, it will be easier to run automated checks to see if each component’s documentation is up to date.
+* Confidence in having access to the single source of truth to ensure that engineers have access to the most relevant up-to-date information
+
+## Risks
+
+* A risk of this strategy is that Platform-wide documentation intended for all VFS developers could be mistakenly added to the Console instead of the Platform Website, making it harder to find. Clear instructions will be needed to avoid this.
```

Comment:
```suggestion
* A risk of this strategy is that Platform-wide documentation intended for all VFS developers could be mistakenly added to the Console instead of the Platform Website, making it harder to find. Platform and VFS Engineers will need clear instructions to avoid this.
```